apps/examples/webserver/webserver_main.c

```diff
@@ -99,12 +102,30 @@ static const char g_httpcnlost[] = "close";
 struct http_server_t *http_server = NULL;
 struct http_server_t *https_server = NULL;
 
+static void http_send_chunk(struct http_client_t *client)
+{
+	char msg_body[MAX_DATA_SIZE + 1] = {0, };
+
+	//add data
+	memset(msg_body, 0, MAX_DATA_SIZE + 1 );
```

Comment:
`msg_body` is already initialized as 0 from line 107. So we don't need to initialize again.
Even if you want to initialize again, we can do just like below.
Then we can save the time consumption.
```
memset(msg_body, 'a', MAX_DATA_SIZE);
msg_body[MAX_DATA_SIZE] = 0;
``

Reply:
done